recipes/varlociraptor/meta.yaml

```diff
@@ -23,18 +21,14 @@ requirements:
   host:
     - gsl
     - libcblas
+    - libcurl
     - openssl
     - zlib
     - bzip2
     - xz
     - clangdev
     - blis
   run:
-    - gsl
-    - openssl
-    - zlib
-    - bzip2
-    - xz
```

Comment:
I removed those because those have `run_exports`, i.e., by listing them in `requirements: host:` they are added to `run` implicitly.